src/components/autocomplete/js/autocompleteDirective.js

```diff
@@ -263,8 +266,40 @@ angular
  * </md-autocomplete>
  * </hljs>
  *
+ * ### Specifying Repeat Mode
+ * You can use `md-mode` to specify whether to use standard or virtual lists for
+ * rendering autocomplete suggestions.
+ * The `md-mode` accepts two values:
+ * - `virtual` (default) Uses `md-virtual-repeat` to render list items. Virtual
+ *    mode requires you to have consistent heights for all suggestions.
```

Comment:
Can you please change `suggestions` here and above/below to `options` for consistency?

Reply:
Sure thing!